### PR TITLE
Format the string literals in /validator directory

### PR DESCRIPTION
**Pull request by steviez**
#### Problem
There are a lots of string literals in this directory, including many for CLI help messages. Any of these strings literals that exceed 100 characters prevent `rustfmt` from running properly.

#### Summary of Changes
- Temporarily adjust `rustfmt.toml` to add `format_strings = true`
- Run `../cargo nightly fmt` within `/validator` directory
- Revert the change to `rustfmt.toml`
- Manual review / tweaks to the changes made
    - The linter tries to fill up the character limit as full as possible. However, some strings are otherwise more readable so adjust those manually / manually revert the strings that were well formatted prior to running the linter

### CODE REVIEW

File: validator/src/cli.rs [hunk 1]
```diff
@@ -294,14 +315,17 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
...
+                     [default: <LEDGER>/accounts_hash_cache]",
+                ),
         )
         .arg(
             Arg::with_name("snapshots")
                 .long("snapshots")
                 .value_name("DIR")
                 .takes_value(true)
-                .help("Use DIR as snapshot location [default: --ledger value]"),
+                .help("Use DIR as snapshot location [default: <LEDGER>/snapshots]"),
```
[the remark this answers is not in the document]
Comment (steviez) [reply validator/src/cli.rs:328]: Ha, should have put it into its' own commit, but figured I'd make that consistent with the others while we were already on the topic today

File: validator/src/bootstrap.rs [hunk 2]
```diff
@@ -490,9 +490,9 @@ fn get_vetted_rpc_nodes(
             Ok(rpc_node_details) => rpc_node_details,
             Err(err) => {
                 error!(
-                    "Failed to get RPC nodes: {err}. Consider checking system clock, removing \
-                     `--no-port-check`, or adjusting `--known-validator ...` arguments as \
-                     applicable"
+                    "Failed to get RPC nodes: {err}. \
+                    Consider checking system clock, removing `--no-port-check`, \
+                    or adjusting `--known-validator ...` arguments as applicable"
```
Comment (CriesofCarrots) [inline validator/src/bootstrap.rs:495]: To Brooks' comment about minimizing the number of special cases handled in the future, I think just keep the `fmt` recommendation here.

File: validator/src/bootstrap.rs [hunk 3]
```diff
@@ -1280,8 +1281,8 @@ fn download_snapshot(
                         warn!(
                             "The snapshot download is too slow, throughput: {} < min speed {} \
                              bytes/sec, but will NOT abort and try a different node as it is the \
-                             only known validator and the --only-known-rpc flag is set. Abort \
-                             count: {}, Progress detail: {:?}",
+                             only known validator and the --only-known-rpc flag is set. \
+                             Abort count: {}, Progress detail: {:?}",
```
Comment (CriesofCarrots) [inline validator/src/bootstrap.rs:1285]: Ditto (just keep the `fmt` recommendation)

File: validator/src/main.rs [hunk 4]
```diff
@@ -1647,8 +1650,8 @@ pub fn main() {
         };
         if limit_ledger_size < DEFAULT_MIN_MAX_LEDGER_SHREDS {
             eprintln!(
-                "The provided --limit-ledger-size value was too small, the minimum value is \
-                 {DEFAULT_MIN_MAX_LEDGER_SHREDS}"
+                "The provided --limit-ledger-size value was too small, \
+                the minimum value is {DEFAULT_MIN_MAX_LEDGER_SHREDS}"
```
Comment (CriesofCarrots) [inline validator/src/main.rs:1654]: Ditto (just keep the `fmt` recommendation)

File: validator/src/main.rs [hunk 5]
```diff
@@ -1621,11 +1621,14 @@ pub fn main() {
...
-             interval (if enabled) \nSnapshot configuration values: \n\tfull snapshot interval: \
-             {} \n\tincremental snapshot interval: {}",
+            "Invalid snapshot configuration provided: snapshot intervals are incompatible. \
+             \n\t- full snapshot interval MUST be a multiple of incremental snapshot interval (if \
+             enabled)\
+             \n\t- full snapshot interval MUST be larger than incremental snapshot \
+             interval (if enabled)\
+             \nSnapshot configuration values:\
+             \n\tfull snapshot interval: {}\
+             \n\tincremental snapshot interval: {}",
```
Comment (CriesofCarrots) [inline validator/src/main.rs:1631]: `fmt` adds a leading space on the lines after the opening quote, which we're losing here. Super nitty, but restore those spaces?

File: validator/src/cli.rs [hunk 6]
```diff
@@ -89,8 +89,9 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .requires("vote_account")
                 .multiple(true)
                 .help(
-                    "Include an additional authorized voter keypair. May be specified multiple \
-                     times. [default: the --identity keypair]",
+                    "Include an additional authorized voter keypair. \
+                     May be specified multiple times. \
+                     [default: the --identity keypair]",
```
Comment (CriesofCarrots) [inline validator/src/cli.rs:94]: The `fmt` suggestion seems fine to me here.

File: validator/src/cli.rs [hunk 7]
```diff
@@ -240,27 +254,33 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
...
             Arg::with_name("health_check_slot_distance")
                 .long("health-check-slot-distance")
                 .value_name("SLOT_DISTANCE")
                 .takes_value(true)
                 .default_value(&default_args.health_check_slot_distance)
-                .help("Report this validator healthy if its latest optimistically confirmed slot \
-                       that has been replayed is no further behind than this number of slots from \
-                       the cluster latest optimistically confirmed slot")
+                .help(
+                    "Report this validator as healthy if its latest, replayed optimistically \
```
Comment (CriesofCarrots) [inline validator/src/cli.rs:280]:
```suggestion
                    "Report this validator as healthy if its latest replayed optimistically \
```
Nit, but the comma hung me up a bit

File: validator/src/bootstrap.rs [hunk 8]
```diff
@@ -905,9 +908,8 @@ fn get_snapshot_hashes_from_known_validators(
         get_snapshot_hashes_for_node,
     ) {
         debug!(
-            "Snapshot hashes have not been discovered from known validators. \
-            This likely means the gossip tables are not fully populated. \
-            We will sleep and retry..."
```
[the remark this answers is not in the document]
Comment (CriesofCarrots) [reply validator/src/bootstrap.rs:910]: Disagree

**Review by CriesofCarrots — COMMENTED** [review]
Just nits, and generally I'm feeling like minimizing the number of special cases (sorry if that's inconsistent with past Tyera!)

Not sure if you did it on purpose, but the last commit is nice to confirm that vanilla format works on the worst of these files now :)

File: validator/src/main.rs (hunk 5 above)
Comment (steviez) [reply validator/src/main.rs:1631]: I might be misunderstanding which item you're referring to, but I think I retained the spaces you're referring to. Here is a screen cap from my editor; the lines as part of the string literal after the first line have a leading space in comparison to the `if` statement directly below:
image

**Review by steviez — COMMENTED** [review]
> Not sure if you did it on purpose, but the last commit is nice to confirm that vanilla format works on the worst of these files now :)

I'd say 50/50 ha; I intentionally rebased to pull in another change, but it didn't occur to me that I'd need to re-run `fmt` to update the change that rebase pulled in, so I was not aware of the failure until this PR went read 😅

**Comment by steviez** [comment]
Last force push was just to rebase in order to (hopefully) address the unrelated CI breakage that went into master; all commits retained

File: validator/src/main.rs (hunk 5 above)
Comment (CriesofCarrots) [reply validator/src/main.rs:1631]: Hmm, strange; yesterday I swore github was not rendering that additional space. But it does indeed look okay now. Sorry for the 🤪

**Review by CriesofCarrots — APPROVED** [review]
:shipit:

**Pull request merged**